mino/minogrpc/certs/disk_test.go

```diff
@@ -0,0 +1,197 @@
+package certs
```

Comment:
these tests depend a lot on the textual information contained in the errors, effectively treating those as constants.

As they're somewhat subject to change (e.g. if we want the error to report the address we were trying to access), could we use constants or sentinel errors to achieve the same objective? I may be missing some pros and cons and am open to discussing it.

Reply:
To discuss, the codebase follows this pattern for errors. It's out of scope of this PR to change, but I guess we can talk about pros and cons.